content/docs/atom-guide/setup-atom-projects.md

```diff
@@ -1,111 +1,66 @@
 ---
-title: "Setting Up Atom Projects"
-description: "Set up Atom projects using the Open 3D Engine"
+linkTitle: "Setup"
+title: "Setting up Atom Renderer"
+description: "Get started with Atom Renderer with an Open 3D Engine project or AtomSampleViewer, Atom's standalone application."
 toc: true
 weight: 200
 ---
 
-Get to know the Atom Renderer and its features by setting up an Atom project with the Open 3D Engine. This tutorial covers how to set up two Atom projects: **Atom Test** and **Atom Sample Viewer**.
-
-- **Atom Test**: This project contains a collection of levels that demonstrates Atom's features in the O3DE Editor.
-- **Atom Sample Viewer**: This project is a standalone application that is intended for developers and engineers who want to test out Atom's individual features. Although you will not run the O3DE Editor, O3DE is still required to build this project.
+Get started with **Atom Renderer** through **Open 3D Engine (O3DE)** by creating a new *O3DE project* or building **Atom Sample Viewer**, Atom's standalone application. 
 
 ## Prerequisites
-Before setting up the Atom projects, you must have the O3DE engine installed on your computer. Follow the instructions in the [Setup](/docs/welcome-guide/setup) section of the Get Started Guide to set up O3DE.
 
+Before setting up a new O3DE project or Atom Sample Viewer, you must install O3DE on your computer. Follow the instructions in [Setting Up O3DE](/docs/welcome-guide/setup) of the Get Started Guide.
 
-## Setting up the Atom projects
 
-This tutorial uses the following project name and directories in the examples:
+## O3DE Project
 
-- O3DE engine directory: `C:\o3de`
-- Atom Test project directory: `C:\AtomTest`
-- Atom Sample Viewer project directory: `C:\AtomSampleViewer`
-- Package directory (created earlier during [setup](/docs/welcome-guide/setup/setup-from-github/)): `C:\o3de-packages`
+Creating an O3DE project provides you the full suite of tools, including **O3DE Editor**, **Material Editor**, and a collection of Gems essential to building a simulation or game. This is a great way to test out Atom's features among other objects and mechanics in a level. 
 
-## Clone project repos
+![Default level prefab](/images/learning-guide/tutorials/environments/create-a-level-C.png)
 
-To get a local copy of the Atom projects, you must clone their repository from GitHub. For best practice, clone the repo into a directory outside of the O3DE repository. 
+### Setup 
 
-Atom Test
-: 
-    ```cmd
-    git clone https://github.com/o3de/o3de-atomtest.git C:\AtomTest
-    ```
+1. Follow the instructions in[Creating a project](/docs/welcome-guide/create/) of the Get Started guide. Atom is built into O3DE via the **Atom Gem**, a core Gem that's enabled in your project by default.
```

Comment:
```suggestion
1. Follow the instructions in[Project Creation](/docs/welcome-guide/create/) in the Get Started guide. After creating an O3DE project, you have access to Atom via the **Atom Gem**. This is a core Gem that's built into O3DE and enabled in your project by default.
```
* _Same cross-reference link changes as for line 13_
* For a clearer throughline from the "follow the instructions" sentence to the sentence about the Atom Gem, I changed "Atom is built into O3DE" to the expanded "After creating an O3DE project, you have access to Atom". (I also split the latter sentence for flow.) I thought that this was clearer, but see what you think.